src/app/(main)/profile/page.tsx

```diff
@@ -101,10 +101,12 @@ export default function Page() {
                   }
                 />
               </div>
-              {formData.password != formData.confirm_password && (
+              {formData.password != formData.confirm_password ? (
                 <p className="text text-sm text-red-500 mb-2">
                   Passwords do not match
                 </p>
+              ) : (
+                <p className="text-transparent text-sm mb-2">Hidden Text</p>
```

Comment:
```suggestion
                <p className="text-transparent text-sm mb-2">&nbsp;</p>
```
Text can still be highlighted : )